makefiles/app_dirs.inc.mk

```diff
@@ -1,13 +1,21 @@
 # fallback so empty RIOTBASE won't lead to "/examples/"
 RIOTBASE ?= .
 
+APPLICATION_DIR_PATTERNS += \
+	$(RIOTBASE)/examples/*/Makefile \
+	$(RIOTBASE)/tests/*/Makefile
+
+ifeq (1, $(ENABLE_STAGING))
+  include $(RIOTBASE)/staging/staging.mk
+  APPLICATION_DIR_PATTERNS += $(STAGING_APPLICATION_MAKEFILES)
```

Comment:
This also shows me that it could be great to have a dedicated filename for the patterns, but it is a different task so no problem to currently have it directly here for now.

Reply:
yes...